src/ip/QosConfig.h

```diff
@@ -240,18 +242,14 @@ class Config
 /// Globally available instance of Qos::Config
 extern Config TheConfig;
 
-/* legacy parser access wrappers */
-#define parse_QosConfig(X)  (X)->parseConfigLine()
-#define free_QosConfig(X)
-#define dump_QosConfig(e,n,X) do { \
-        char temp[256]; /* random number. change as needed. max config line length. */ \
-        (X).dumpConfigLine(temp,n); \
-            storeAppendPrintf(e, "%s", temp); \
-    } while(0);
-
 } // namespace Qos
 
 } // namespace Ip
 
+/* legacy parser access wrappers */
+inline void parse_QosConfig(Ip::Qos::Config * theConfig) { theConfig->parseConfigLine(); }
```

Comment:
We tend to use `theX` format for class data members (and only in legacy code; modern code uses `x_` instead).

```suggestion
inline void parse_QosConfig(Ip::Qos::Config * const c) { c->parseConfigLine(); }
```